editor/src/components/canvas/controls/grid-controls.tsx

```diff
@@ -648,35 +649,45 @@ export const GridControls = controlForStrategyMemoized(() => {
         {/* grid lines */}
         {grids.map((grid) => {
           const placeholders = Array.from(Array(grid.cells).keys())
+          let style: CSSProperties = {
+            position: 'absolute',
+            top: grid.frame.y,
+            left: grid.frame.x,
+            width: grid.frame.width,
+            height: grid.frame.height,
+            display: 'grid',
+            gridTemplateColumns: getNullableAutoOrTemplateBaseString(grid.gridTemplateColumns),
+            gridTemplateRows: getNullableAutoOrTemplateBaseString(grid.gridTemplateRows),
+            backgroundColor:
+              activelyDraggingOrResizingCell != null ? colorTheme.primary10.value : 'transparent',
+            border: `1px solid ${
+              activelyDraggingOrResizingCell != null ? colorTheme.primary.value : 'transparent'
+            }`,
+            padding:
+              grid.padding == null
+                ? 0
+                : `${grid.padding.top}px ${grid.padding.right}px ${grid.padding.bottom}px ${grid.padding.left}px`,
+          }
+
+          // Gap needs to be set only if the other two are not present or we'll have rendering issues
+          // due to how measurements are calculated.
```

Comment:
minor - but if we're always only allowing to play with `rowGap` and `columnGap` - why do we need to refer to `gap` at all?
meaning why not have:
```
style.rowGap = grid.rowGap || grid.gap
style.columnGap = grid.columnGap || grid.gap
```
since we don't refer to `gap` anywhere anyway?

Reply:
we need to, because `gap` can override the other twos